app/views/items/favorite.js.erb

```diff
@@ -0,0 +1,2 @@
+//$(".favBtn")[0].innerHTML = "Product Created";
+alert("here")
```

Comment:
?

Reply:
hahaha this was probably for debugging --> better to delete this afterwards @njackson1-code

Reply:
also wait... shouldnt this be located in `app/javascript`  and not in `app/views/`?